docs/releases/4.1.txt

```diff
@@ -251,6 +251,11 @@ Management Commands
 * The new :djadmin:`optimizemigration` command allows optimizing operations for
   a migration.
 
+* :djadmin:`makemessages` command no longer processes invalid locales specified
+  using :option:`makemessages --locale` option, when they contain hyphens
+  (``'-'``) between the language and country or when the country code is
+  lowercase.
+
```

Comment:
Wasn't the point that we're enabling the custom locales? 🤔

This looks like an _"As a consequence of this, ..."_ extra?

Reply:
I also changed the release notes

Reply:
Use double-back ticks for code, such as ``nl_NL-x-informal``. 

(I can resolve when reviewing if it doesn't work for you immediately.)

Reply:
@carltongibson I resolved the `black` issues and force-pushed it. If you would fix the spelling that would be great. Thank you.